sys/include/crypto/modes/cbc.h

```diff
@@ -36,9 +37,13 @@ extern "C" {
  * @param input_len  length of the input data
  * @param output     pointer to allocated memory for encrypted data. It has to
  *                   be of size data_len + BLOCK_SIZE - data_len % BLOCK_SIZE.
+ *
+ * @return            <0 on error
+ * @return            0 on success
```

Comment:
Not sure this is needed, because 0 bytes are consumed on success so the next return would be enough